cluster/gce/gci/configure-helper.sh

```diff
@@ -2089,10 +2089,14 @@ function start-fluentd-resource-update {
   wait-for-apiserver-and-update-fluentd &
 }
 
-# Update {{ container-runtime }} with actual container runtime name.
+# Update {{ container-runtime }} with actual container runtime name,
+# and {{ container-runtime-endpoint }} with actual container runtime
+# endpoint.
 function update-container-runtime {
-  local -r configmap_yaml="$1"
-  sed -i -e "s@{{ *container_runtime *}}@${CONTAINER_RUNTIME_NAME:-docker}@g" "${configmap_yaml}"
+  local -r file="$1"
+  sed -i -e "s@{{ *container_runtime *}}@${CONTAINER_RUNTIME_NAME:-docker}@g" "${file}"
+  container_runtime_endpoint=${CONTAINER_RUNTIME_ENDPOINT:-"unix:///var/run/dockershim.sock"}
```

Comment:
Make this variable local (and readonly too).

Do both substitutions using a single `sed` command:

```shell
  sed -i \
      -e "s@{{ *container_runtime *}}@${CONTAINER_RUNTIME_NAME:-docker}@g" \
      -e "s@{{ *container_runtime_endpoint *}}@${container_runtime_endpoint#unix://}@g" \
      "${file}"
```

Also remove the quotes around `"unix://"` as they do nothing... Removing them means you no longer have "nested quotes."

Reply:
Done